LocalPackages/PreferencesUI/Package.swift

```diff
@@ -0,0 +1,23 @@
+// swift-tools-version: 5.9
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "PreferencesUI",
```

Comment:
New PreferencesUI package to be able to share some common components with more ease.